bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/BallerinaErrors.java

```diff
@@ -60,9 +60,14 @@
     public static final String GENERATE_PKG_START = "___start_";
     public static final String GENERATE_PKG_STOP = "___stop_";
     public static final String GENERATE_OBJECT_CLASS_PREFIX = ".$value$";
+    public static final String IS_STRING_VALUE_PROP = "ballerina.bstring";
+    public static final boolean USE_BSTRING = System.getProperty(IS_STRING_VALUE_PROP) != null;
```

Comment:
Seems getting this system property is repeated multiple times. Can we have this at one place and refer from wherever needed?

Reply:
I thought I would be cleaning up later. Couldn't think of top level place for all the modules? Any suggestions? Or we can leave it as I would be removing it after moving to a new branch.